ts/Extensions/Breadcrumbs.ts

```diff
@@ -0,0 +1,898 @@
+            formatter?: BreadcrumbsButtonsFormatter;
+            position?: BreadcrumbsAlignOptions;
+            separator?: string;
+            showOnlyLast?: boolean;
+            style?: CSSObject;
+            useHTML?: boolean;
+            zIndex?: number;
+        }
+        interface BreadcrumbsAlignOptions {
+            align?: AlignValue;
+            verticalAlign?: VerticalAlignValue;
+            alignByTranslate?: boolean;
+            x: number;
+            y: number;
+        }
+        interface BreadcrumbsClickCallbackFunction {
+            (e: Event, breadcrumbs: Breadcrumbs): (boolean|undefined);
+        }
+        interface BreadcrumbsButtonsEventsOptions {
+            click?: BreadcrumbsClickCallbackFunction;
+        }
+        interface BreadcrumbsButtonsFormatter {
+            (
+                breadcrumb: Array<any>,
+                breadcrumbs: Breadcrumbs
+            ): (string);
+        }
+        interface Options {
+            breadcrumbs?: BreadcrumbsOptions;
+        }
+        interface ChartLike {
+            breadcrumbs?: Breadcrumbs;
+        }
+        interface LangOptions {
+            mainBreadCrumb?: string;
+        }
+        class Breadcrumbs {
+            public constructor(chart: Chart, userOptions: Highcharts.Options);
+
+            public breadcrumbsGroup: SVGElement;
+            public breadcrumbsList: Array<Array<unknown>>
+            public chart: Chart;
+            public isTreemap: boolean;
+            public level: number;
+            public options: BreadcrumbsOptions;
+
+            public alignGroup(): void;
+            public createList(e?: any): void;
+            public destroy(this: Breadcrumbs, lastVisibleLevel: number|undefined): void;
+            public draw(): void;
+            public multipleDrillUp(numb: number): void;
+            public init(chart: Chart, userOptions: Highcharts.Options): void;
+            public redraw(): void;
+            public renderButton(breadcrumb: Array<any>, posX: number, posY: number): SVGElement|void;
+            public renderSeparator(posX: number, posY: number): SVGElement|void;
+            public update(userOptions: Highcharts.Options, redrawFlag: boolean): void;
+        }
+    }
+}
+
+// Add language support.
+extend(
+    defaultOptions.lang,
+    /**
+     * @optionparent lang
+     */
+    {
+        /**
+         * @since    next
+         * @product  highcharts
+         */
+        mainBreadCrumb: 'Main'
+    }
+);
+
+extend(
+    defaultOptions,
+    {
+        /**
+         * Options for breadcrumbs
+         *
+         * @since   next
+         * @product highcharts
+         */
+        breadcrumbs: {
+            /**
+            * The default padding for each button and separator.
+            *
+            * @type      {boolean}
+            * @since     next
+            */
+            buttonPadding: 5,
+
+            /**
+            * Show only last button instead of whole drillUp tree.
```

Comment:
Just an idea:
Instead of `showOnlyLast`, let's call it `showFullPath` or `showAboslutePath`. The option would be self-describing.

That reminds me, how about long paths? For example this: http://jsfiddle.net/BlackLabel/5hctznvw/2/

Reply:
I agree. I prefer `showFullPath`.